src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeCleanup/CodeCleanupOptionsProviders.cs

```diff
@@ -23,12 +24,23 @@ public static CodeCleanupOptions GetCodeCleanupOptions(this IOptionsReader optio
             DocumentFormattingOptions = options.GetDocumentFormattingOptions(),
         };
 
-#if !CODE_STYLE
+#pragma warning disable CS1998 // Async method lacks 'await' operators and will run synchronously
     public static async ValueTask<CodeCleanupOptions> GetCodeCleanupOptionsAsync(this Document document, CancellationToken cancellationToken)
+#pragma warning restore CS1998 // Async method lacks 'await' operators and will run synchronously
     {
+#if CODE_STYLE
+        return GetDefault(document.Project.GetExtendedLanguageServices().LanguageServices);
+#else
         var configOptions = await document.GetAnalyzerConfigOptionsAsync(cancellationToken).ConfigureAwait(false);
         return configOptions.GetCodeCleanupOptions(document.Project.Services, document.AllowImportsInHiddenRegions());
-    }
 #endif
```

Comment:
wasn't sure hwo to implement this one. @tmat is there any way for us to get GetAnalyzerConfigOptionsAsync in teh codefix layer?  if so, we could have ll this code work the same way in sdk and VS.

Reply:
figured this out.